Sources/PackageManagerDocs/Documentation.docc/PackageCollections.md

```diff
@@ -4,18 +4,11 @@ Learn to create, publish and use Swift package collections.
 
 ## Overview
 
-Package collections, introduced by [SE-0291](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0291-package-collections.md), are
-curated lists of packages and associated metadata that can be imported
-by SwiftPM to make discovery of existing packages easier. 
-They are authored as static JSON documents 
-and can be published to the web or distributed to local file systems. 
-
-Educators and community influencers can publish
-package collections to go along with course materials or blog posts, removing the friction of using
-packages for the first time and the cognitive overload of deciding which packages are useful for
-a particular task. 
-Enterprises may use collections to narrow the decision space for their internal
-engineering teams, focusing them on a trusted set of vetted packages.
+Package collections, introduced by [SE-0291](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0291-package-collections.md), are curated lists of packages and associated metadata that package manager can import to make discovery of existing packages easier. 
+Author a package collection as a static JSON document and publish it to the web or distribute it on a local file system.
+
+Educators and community influencers can publish package collections to go along with course materials or blog posts, making it easier for their readers to use packages for the first time, or choosing which packages to use for a particular task. 
+Enterprises may use collections to provide a trusted set of packages, or a collection of packages consistently used by a team.
```

Comment:
```suggestion
Educators and community influencers can publish package collections to go along with course materials or blog posts, making it easier for their readers to use packages for the first time, or choosing which packages to use for a particular task.
Enterprises may use collections to provide a trusted set of packages, or a collection of packages consistently used by a team.
Author a package collection as a static JSON document and publish it to the web or distribute it on a local file system.
```

What do you think of this as one paragraph? I think the "author this by..." stuff may read a little better after we've described the two use cases.

Reply:
I think it reads a lot better this way! I'll upload a change for this :)